src/lib/Search/Legacy/Content/WordIndexer/Gateway/DoctrineDatabase.php

```diff
@@ -30,7 +30,7 @@ class DoctrineDatabase extends Gateway
     protected $connection;
 
     /**
-     * SPI Content Type Handler.
+     * SPI Content type Handler.
```

Comment:
Since you touched it... :upside_down_face: 
```suggestion
     * Persistence content type handler.
```
that's the correct term. Persistence in this context could be skipped because it can be inferred from DoctrineDatabase class itself.